src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./Introduce.scss";
+import { Link, useNavigate } from "react-router-dom";
+
+const Introduce = () => {
+  const navigate = useNavigate();
+
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+
+  const handleButton = () => {
+    console.log("button 태그 동작");
+    navigate("/product");
+  };
+
+  return (
+    <div className="Introduce">
```

Comment:
```suggestion
    <div className="introduce">
```

최상위 부모 태그 className에 컴포넌트 이름을 잘 적용해 주셨네요!
다만 첫글자가 대문자인 Pascal case는 컴포넌트의 이름에만 적용되는 규칙입니다!

className은 소문자로 시작해주세요!